RoeiVerenigingWPF/Pages/MainPage.xaml

```diff
@@ -3,9 +3,101 @@
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="clr-namespace:RoeiVerenigingWPF.helpers"
       mc:Ignorable="d"
-      Title="MainPage" Height="900" Width="800">
+      Title="MainPage" Height="900" Width="1000">
+    <!--Events-->
     <Grid>
-        <Frame Name="StatisticsFrame"></Frame>
+        <Border CornerRadius="15" BorderThickness="5" BorderBrush="{x:Static local:CustomColors.OutsideBorderColor}">
+            <Border CornerRadius="10" BorderThickness="8"
+            BorderBrush="{x:Static local:CustomColors.MainBackgroundColor}">
+                <Grid Height="900" Width="1000" Background="{x:Static local:CustomColors.MainBackgroundColor}"
+              HorizontalAlignment="Center"
+              VerticalAlignment="Center">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="auto"/>
+                        <RowDefinition/>
+                        <RowDefinition Height="50"/>
+                    </Grid.RowDefinitions>
+
+                    <Label Grid.Row="0"
+                    Content="Evenements" HorizontalContentAlignment="Center" VerticalAlignment="Top"
+                    Margin="0,30,0,0"
+                    FontFamily="Franklin Gothic Medium" FontSize="36"
+                    Foreground="{x:Static local:CustomColors.HeaderColor}" Grid.ColumnSpan="2" />
+
+                    <Grid Grid.Row="1">
+                        <ListView ItemsSource="{Binding _events}" Grid.IsSharedSizeScope="True"
```

Comment:
waarom is hier gekozen voor een listview ipv een scrollviewer met een stackpanel erin?